src/amoc_config/amoc_config_validation.erl

```diff
@@ -0,0 +1,60 @@
+%%==============================================================================
+%% Copyright 2020 Erlang Solutions Ltd.
+%% Licensed under the Apache License, Version 2.0 (see LICENSE file)
+%%==============================================================================
+%% this module is responsible for processing scenario configuration
+%%==============================================================================
+-module(amoc_config_validation).
```

Comment:
We are using the term `validation` here, but `verification` elsewhere, could we be more consistent with naming?

Reply:
that is the only inconsistency in naming, i just didn't want to change file names. git doesn't handle it well. but i can do it if you insist.

Reply:
Isn't it a new file? Anyway, this is a minor thing, not blocking the approval of this PR.